### DISCUSSION TITLE
Bun v1.0.22

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#8074] [Announcements]
To install Bun v1.0.22:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.22:
```bash
bun upgrade
```

**[Read Bun v1.0.22's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.22)**

## Thanks to 22 contributors!

Thank you to all the contributors to this release of Bun, including the new 13 contributors who submitted their first pull request!

- [aarvinr](https://github.com/@aarvinr)
- [asomethings](https://github.com/@asomethings)
- [coratgerl](https://github.com/@coratgerl)
- [guest271314](https://github.com/@guest271314)
- [gvilums](https://github.com/@gvilums)
- [Hanaasagi](https://github.com/@Hanaasagi)
- [hborchardt](https://github.com/@hborchardt)
- [huseeiin](https://github.com/@huseeiin)
- [jakeg](https://github.com/@jakeg)
- [Jarred-Sumner](https://github.com/Jarredd-Sumner)
- [jdalton](https://github.com/@jdalton)
- [karmabadger](https://github.com/@karmabadger)
- [knightspore](https://github.com/@knightspore)
- [lino-levan](https://github.com/lino-levan)
- [markusn](https://github.com/@markusn)
- [MatricalDefunkt](https://github.com/@MatricalDefunkt)
- [morisk](https://github.com/@morisk)
- [nektro](https://github.com/@nektro)
- [otgerrogla](https://github.com/@otgerrogla)
- [paperdave](https://github.com/@paperdave)
- [sitiom](https://github.com/@sitiom)
- [ThatOneCalculator](https://github.com/@ThatOneCalculator)
- [twlite](https://github.com/@twlite)

You can also read the [full changelog](https://github.com/oven-sh/bun/compare/bun-v1.0.21...bun-v1.0.22).

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.21...bun-v1.0.22

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.22'>Bun v1.0.22</a>.</em>

**Comment by Geisson19** [comment]
Contributors links are broken btw (they have an extra `@` in between)
